lighthouse-cli/test/cli/bin-test.js

```diff
@@ -170,6 +170,26 @@ describe('CLI bin', function() {
     });
   });
 
+  describe('extraCookies', () => {
+    it('should convert extra cookies to object', async () => {
+      // @ts-ignore - see TODO: in bin.js
+      cliFlags = {...cliFlags, extraCookies: '[{"name":"foo", "value": "bar", "url": "http://localhost"}]'};
+      await bin.begin();
+
+      expect(getRunLighthouseArgs()[1]).toHaveProperty('extraCookies', [{'name': 'foo', 'value': 'bar', 'url': 'http://localhost'}]);
+    });
+
+    it('should read extra cookies from file', async () => {
+      const headersFile = require.resolve('../fixtures/extra-headers/valid.json');
```

Comment:
seems like you mean to use the cookies one instead?

Reply:
Fixed by f4e0a49